EmojicodeCompiler/Lexer.cpp

```diff
@@ -142,6 +143,29 @@ TokenStream lex(const char *path) {
                         /* End of variable */
                         nextToken = true;
                     }
+					else if (c == 0xFE0F) {
+						// Variation Selector 16
+						possiblyIdentifier = true;
+						continue;
+					}
+					else if (possiblyIdentifier) {
+						if (c == 0x20E3) {
```

Comment:
I really dislike duplicate code. Can you probably try to move this into a function?